### PR TITLE
Minor typo, punctuation update

### PR DESCRIPTION
**Pull request by andrewpollack**

### CODE REVIEW

File: src/structure.md [hunk 1]
```diff
@@ -15,8 +15,9 @@ days:
 ## Format
 
 The course is interactive and your questions will drive our exploration of Rust!
+You may ask questions are start discussions by filing issues [here](https://github.com/google/comprehensive-rust/issues).
```
Comment (qwandor) [inline src/structure.md:18]: s/are/or/

Comment (andrewpollack) [reply src/structure.md]: Good eye -- ty!

File: src/structure.md [hunk 2]
```diff
@@ -15,8 +15,9 @@ days:
 ## Format
 
 The course is interactive and your questions will drive our exploration of Rust!
+You may ask questions or start discussions by filing issues [here](https://github.com/google/comprehensive-rust/issues).
```
Comment (mgeisler) [inline src/structure.md:18]: This page (ehem "slide") is shown during the class, so I would prefer to not have this part here. I've added links to the GitHub discussions in the README and on the first welcome page — I think they work better there.

Comment (andrewpollack) [reply src/structure.md]: Apologies -- I missed those inclusions on the first pass. Removed this portion from the PR!

**Pull request merged**